charts/atlantis/values.yaml

```diff
@@ -1,4 +1,4 @@
-## -------------------------- ##
+0.18.2## -------------------------- ##
```

Comment:
same here

Reply:
oh, i used the vscode replace. let me correct it

Reply:
should be good now (codespace is still not ideal for such operations 😮‍💨 )